src/components/ItemDataPanel.jsx

```diff
@@ -1,8 +1,16 @@
 import{ PropTypes } from 'prop-types';
 import RoundedButton from './buttons/RoundedButton';
+import {useNavigate} from 'react-router-dom';
 
 function ItemDataPanel(props) {
   const { title, organizer, description, price, date, city } = props.concert;
+  const navigate = useNavigate();
+
+  const handleReserveClick = () => {
+    console.log("Reserve button clicked")
```

Comment:
- [ ] Please delete this log to the console, using console.log is good for debugging your work in the development environment but not in the production environment.

Reply:
Thanks for pointing that out, I'll do it right now!

Reply:
Removed! @danifromecuador